shared/translations/languages/de.json

```diff
@@ -315,7 +315,7 @@
     "Active (Green):  When the collateralization ratio of a vault is more than 1.5x the minimum collateralization ratio.": "Aktiv (Grün): Wenn die Besicherungsquote eines Vaults mehr als das 1,5-Fache der Mindestbesicherungsquote beträgt.",
     "In liquidation: When a vault's collateralization ratio falls below the minimum requirement.": "In Liquidation: Wenn die Besicherungsquote eines Vaults unter die Mindestanforderung fällt.",
     "Halted: When any token in the vault (collateral or loan) has fluctuated more than 30% in the past hour.": "Gestoppt: Wenn ein Token im Vault (Sicherheit oder Kredit) in der letzten Stunde um mehr als 30% geschwankt hat.",
-    "How are interests calculated for loan?": "Wie werden die Zinsen für das Darlehen berechnet?",
+    "How are interests calculated for loan?": "Wie werden die Zinsen für  Darlehen berechnet?",
```

Comment:
Is there a double space here?

Reply:
You are right, no double space. Changed it.